slick/src/main/scala/slick/jdbc/DB2Profile.scala

```diff
@@ -180,6 +185,70 @@ trait DB2Profile extends JdbcProfile {
       override def sqlTypeName(sym: Option[FieldSymbol]) = "CHAR(1)"
       override def valueToSQLLiteral(value: Boolean) = if(value) "1" else "0"
     }
+
+    class InstantJdbcType extends super.InstantJdbcType {
+      override def sqlTypeName(sym: Option[FieldSymbol]) = "TIMESTAMP(6) WITH TIME ZONE"
```

Comment:
Why would Instant have a time zone? Instant is just time steps since `1970-01-01T00:00:00Z`, i.e. it is always Zulu time